docs/users-guide/field-types.md

```diff
@@ -0,0 +1,98 @@
+# Field types in Metabase
+
+Understanding what field types are and how they function in Metabase will help you become more familiar with your own data and get the most out of Metabase.
```

Comment:
```suggestion
```
Think we can skip the preamble; if they're on this page, they want to know about field types.